meson.build

```diff
@@ -199,6 +199,22 @@ gumjs_extra_deps = []
 gumjs_extra_requires = []
 gumjs_extra_libraries = []
 
+have_gir = false
+
+if cc.has_function('g_thread_set_callbacks', dependencies: [glib_dep])
+  cdata.set('HAVE_FRIDA_GLIB', 1)
+else
+  elf_dep = dependency('libelf')
+  extra_deps += [elf_dep]
```

Comment:
I think since we were using aggressive static linking a peer had the same dependency and it pulled it in. On my system, I was getting linker errors to symbols inside `libelf`. This PR adds it conditionally because the Frida toolchain doesn't provide a `libelf` pkg-config, so adding a `dependency('libelf')` will throw a missing dependency error.